Repository/Api/EditableRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Resource\Repository\Api;
+
+use InvalidArgumentException;
+use Puli\Repository\Api\EditableRepository;
+use Puli\Repository\Api\UnsupportedLanguageException;
+
+/**
+ * Extends the Puli editable repository to implement the as-of-yet not
+ * implemented features.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+interface EditableRepository extends EditableRepository
```

Comment:
Hmm, yeah. Somehow strange that that even works.

On Thu, Jun 02, 2016 at 07:56:46AM -0700, Maximilian Berghoff wrote:
>    In [1]Repository/Api/EditableRepository.php:
> 
>  > + * file that was distributed with this source code.
>  > + */
>  > +
>  > +namespace Symfony\Cmf\Component\Resource\Repository\Api;
>  > +
>  > +use InvalidArgumentException;
>  > +use Puli\Repository\Api\EditableRepository;
>  > +use Puli\Repository\Api\UnsupportedLanguageException;
>  > +
>  > +/**
>  > + * Extends the Puli editable repository to implement the as-of-yet not
>  > + * implemented features.
>  > + *
>  > + * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
>  > + */
>  > +interface EditableRepository extends EditableRepository
> 
>    I think you should alias the second one.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly, [2]view it on GitHub, or [3]mute the thread.
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/resource/pull/23#discussion_r65555686
>    2. https://github.com/symfony-cmf/resource/pull/23/files/c12c5cf7b1de0cbf2bcb789fffc4610038aa5567#r65555686
>    3. https://github.com/notifications/unsubscribe/AAgZccdL1mCPtwD8BOrwyN4p1roz7Bxmks5qHu8ugaJpZM4IsYhH
>    4. https://github.com/symfony-cmf/resource/pull/23/files/c12c5cf7b1de0cbf2bcb789fffc4610038aa5567#r65555686